.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "cvs-nop"]
 	path = cvs-nop
 	url = https://github.com/dvsa/cvs-nop
-	branch = CVSB-19155
+	branch = feature/CVSB-19421
```

Comment:
I think we might need to come up with a way to inject our branch as an environmental variable so it picks up the changes of the relevant branch, or changes we want to make.
I am thinking of cases where the developer or remote has the cvs-nop updated (and their schema) which could lead to situation where the DB loaded locally is not what we expect.
What about having a script that generates this file and injects the branch just after we have cloned and installed the repo?
Then once the file is generated we would just have to run git submodule update --init --recursive ?

Reply:
As discussed, created a ticket https://jira.dvsacloud.uk/browse/CVSB-19447 and resolving